[2024-04-16T10:35:56.503+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-16T10:35:56.563+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: priority_one_number_two_dag_v1.generate_random_number scheduled__2024-02-24T00:00:00+00:00 [queued]>
[2024-04-16T10:35:56.596+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: priority_one_number_two_dag_v1.generate_random_number scheduled__2024-02-24T00:00:00+00:00 [queued]>
[2024-04-16T10:35:56.598+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-16T10:35:56.727+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): generate_random_number> on 2024-02-24 00:00:00+00:00
[2024-04-16T10:35:56.821+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=501) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-16T10:35:56.866+0000] {standard_task_runner.py:63} INFO - Started process 513 to run task
[2024-04-16T10:35:56.846+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'priority_one_number_two_dag_v1', 'generate_random_number', 'scheduled__2024-02-24T00:00:00+00:00', '--job-id', '300', '--raw', '--subdir', 'DAGS_FOLDER/priority_one_number_two_dag.py', '--cfg-path', '/tmp/tmp7sm8p6y2']
[2024-04-16T10:35:56.879+0000] {standard_task_runner.py:91} INFO - Job 300: Subtask generate_random_number
[2024-04-16T10:35:57.352+0000] {task_command.py:426} INFO - Running <TaskInstance: priority_one_number_two_dag_v1.generate_random_number scheduled__2024-02-24T00:00:00+00:00 [running]> on host eaff2d74d54d
[2024-04-16T10:35:57.677+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ika' AIRFLOW_CTX_DAG_ID='priority_one_number_two_dag_v1' AIRFLOW_CTX_TASK_ID='generate_random_number' AIRFLOW_CTX_EXECUTION_DATE='2024-02-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-24T00:00:00+00:00'
[2024-04-16T10:35:57.680+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-16T10:35:57.926+0000] {python.py:237} INFO - Done. Returned value was: [19, 7, 1, 37, 43, 7, 16, 16, 11, 5, 36, 5, 15, 10, 16, 23, 46, 38, 19, 1, 34, 18, 22, 40, 7]
[2024-04-16T10:35:57.928+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-16T10:35:58.025+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=priority_one_number_two_dag_v1, task_id=generate_random_number, execution_date=20240224T000000, start_date=20240416T103556, end_date=20240416T103558
[2024-04-16T10:35:58.115+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-16T10:35:58.475+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-16T10:35:58.499+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
